packages/components/src/confirm-dialog/styles.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * External dependencies
+ */
+import { css } from '@emotion/react';
+
+/**
+ * The z-index for ConfirmDialog is being set here instead of in
+ * packages/base-styles/_z-index.scss, because this component uses
+ * emotion instead of sass.
+ *
+ * ConfirmDialog needs this higher z-index to ensure it renders on top of
+ * any parent Popover component.
+ */
+export const wrapper = css`
+	z-index: 1000001;
+`;
```

Comment:
As explained earlier, the CSS rule resulting from this snippet may not be specific enough to override the default `z-index`. A way to fix this is to use the `&&` syntax to increase the rule's specificity:

```suggestion
export const wrapper = css`
	&& {
		z-index: 1000001;
	}
`;
```

Reply:
I see what you mean, and this fix helps. Odd that the CSS rule appeared specific enough in my initial tests before submitting this PR, the Popover was definitely rendering under the ConfirmDialog at that point. After retesting though, it definitely needs this added specificity, thank you!